### PR TITLE
DIO: check that a bucket is not public

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/checks/resource/digitalocean/SpacesBucketPublicRead.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from checkov.terraform.checks.resource.base_resource_negative_value_check import BaseResourceNegativeValueCheck
+from typing import List, Any
+
+
+class SpaceBucketPublicRead(BaseResourceNegativeValueCheck):
+    def __init__(self):
+        name = "Ensure the Spaces bucket is private"
+        id = "CKV_DIO_3"
+        supported_resources = ['digitalocean_spaces_bucket']
+        categories = [CheckCategories.BACKUP_AND_RECOVERY]
```
Comment (gruebel) [inline checkov/terraform/checks/resource/digitalocean/SpacesBucketPublicRead.py:11]:
```suggestion
        categories = [CheckCategories.NETWORKING]
```
probably fits better?

File: tests/terraform/checks/resource/digitalocean/example_SpaceBucketPublicRead/main.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  name   = "public_space"
+  region = "nyc3"
+  acl    = "public-read"
+}
+
+
+resource "digitalocean_spaces_bucket" "pass" {
+  name   = "public_space"
+  region = "nyc3"
+  acl    = "private"
```
Comment (gruebel) [inline tests/terraform/checks/resource/digitalocean/example_SpaceBucketPublicRead/main.tf:12]: the default value is `private` right?

Comment (JamesWoolfenden) [reply tests/terraform/checks/resource/digitalocean/example_SpaceBucketPublicRead/main.tf:12]: it is.

File: checkov/terraform/checks/resource/digitalocean/SpacesBucketPublicRead.py (hunk 1 above)
Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/digitalocean/SpacesBucketPublicRead.py:11]: I dunno. S3S3PublicACLRead uses GENERAL_SECURITY, maybe thats better?

Comment (gruebel) [reply checkov/terraform/checks/resource/digitalocean/SpacesBucketPublicRead.py:11]: yeah, sounds also good 👍

**Review by gruebel — APPROVED** [review]
🍺

**Pull request merged**